### ISSUE TITLE
Ability to change buildables owner and group owner.

### ISSUE THREAD

**Issue by Adamv21** [SmartlyDressedGames/Unturned-3.x-Community#464]
Right now (from what i'm aware) you can't change the owner and group owner of all barricades / structures. It's possible to change owner of an barricade with an InteractableStorage component attached to it (I think but not tested all other things with a state that can be updated too). What i'm asking is basiaclly maybe a steamcall to change owner info of a buildable (server sided ofc).

Hope you can implement something like this soon!

**Comment by aviadmini** [comment]
Shame on you, it's possible and not even that hard

**Comment by Adamv21** [comment]
Well, love you too I guess. Wasn't aware it was. I doubt someone like you would shard any valid info on how to but atleast know I know it's possible. Time to look some more into it.

**Comment by Johnanater** [comment]
Probably not, because selling plugins with this "not even that hard" stuff is quite profitable.

If you find out how, be sure to post it here :)

**Comment by aviadmini** [comment]
Well I went a bit further than just that. 

Anyway it's a good exercise to figure it out instead of making Nelson make it for ya

**Comment by SDGNelson** [comment]
Saving owner for all buildables was done a while after owners were first used, so right now they're kept in two places: some with lock checks e.g. doors usually have that info in the first 16 bytes of their state, and BarricadeData has the IDs used for salvaging + hiding health.

Were you hoping to be able to change salvage owner, or owner for the doors and stuff?

**Comment by Adamv21** [comment]
Yeah a steamcall to modify the owner of barricade would be amazing.

**Comment by SDGNelson** [comment]
Will likely add a method to easily change owner for the next update! Not in the unity update for today yet though
